src/data/events.js

```diff
@@ -2,8 +2,8 @@ import { writable } from 'svelte/store'
 import { GET, POST, PUT } from './api'
 import { onClear } from './storage'
 
-export const events = writable(null)
-export const loading = writable(false)
+export const events = writable([])
+export const loading = writable(true)
```

Comment:
I would prefer not to solve it this way. At initialization, the `events` store is not in a loading state, so to say that `loading` is `true` is not correct. But if you're hitting a wall, we can leave it (add a TODO comment to flag the issue) or pair on it.